fitlins/cli/run.py

```diff
@@ -96,10 +96,11 @@ def get_parser():
                         help="use BOLD files with the provided description label")
 
     g_prep = parser.add_argument_group('Options for preprocessing BOLD series')
-    g_prep.add_argument('-s', '--smoothing', action='store', metavar="TYPE:FWHM",
+    g_prep.add_argument('-s', '--smoothing', action='store', metavar="TYPE:FWHM:LEVEL",
```

Comment:
I would prefer the levels to be named, e.g. `run`, `session`, etc. So instead of `-s iso:5:1`, it would be `-s run:iso:5`.

It doesn't seem likely that we'll want to smooth multiple times, does it?